R/fda-table_8.R

```diff
@@ -0,0 +1,56 @@
+# Pre-processing
+library(dplyr)
+library(scda)
+library(tern)
+library(rlistings)
+
+# Deaths
+adae <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adae") %>%
+  filter(SAFFL == "Y" & AESDTH == "Y") %>%
+  select(
+    STUDYID, USUBJID, ACTARM, ACTARMCD, AGE, SEX, AETERM, AEDECOD, ASTDY, AESDTH, AEOUT, DTHADY
+  ) %>%
+  mutate(AGESEX = paste0(AGE, "/", SEX)) %>%
+  distinct(USUBJID, AETERM, AEDECOD, DTHADY, .keep_all = TRUE)
+
+# Dosing
+adex <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adex") %>%
+  filter(PARAMCD == "TDOSE") %>%
+  select(USUBJID, PARAMCD, PARAM, AVAL, AVALU, TRTSDT, TRTEDT) %>%
+  mutate(
+    DOSDUR = (TRTEDT - TRTSDT + 1),
+    DOSAGE = paste0(AVAL, " ", AVALU)
+  )
+
+# Join
+tbl_join <-
+  adae %>%
+  left_join(adex, by = "USUBJID") %>%
+  select(ACTARM, USUBJID, AGESEX, DOSAGE, DOSDUR, DTHADY, AEDECOD, AETERM) %>%
```

Comment:
Same variable replacement again
```suggestion
  select(ACTARM, USUBJID, AGESEX, DOSAGE, DOSDUR, DTHADY, DTHCAUS, DTHCAT) %>%
```